SZMentionsSwift/Classes/MentionListener.swift

```diff
@@ -147,10 +147,10 @@ public class MentionListener: NSObject {
         hideMentions: @escaping () -> Void,
         didHandleMentionOnReturn: @escaping () -> Bool,
         showMentionsListWithString: @escaping (String, String) -> Void
-    ) {
-        mentionTextAttributes = mentionAttributes ?? [Attribute(name: NSAttributedStringKey.foregroundColor.rawValue,
+        ) {
+        mentionTextAttributes = mentionAttributes ?? [Attribute(name: NSAttributedString.Key.foregroundColor.rawValue,
```

Comment:
I think you can update the `Attribute` type to take an `NSAttributedString.Key` and avoid this specification entirely. It should also allow for line 293 below to be removed entirely. Previously, name was a `String` in order to allow for older versions of Swift.